src/device_config/config_parser.c

```diff
@@ -96,9 +96,13 @@ void parse_config() {
       hal_gpio_init(pin, 1, pull);
 
       buttons[buttons_cnt].pin = pin;
-      buttons[buttons_cnt].long_press_duration_ms = 2000;
+      // Reset buttons get 5s long press to prevent accidental resets
```

Comment:
This is a dedicated reset button, often hidden inside the wall, so there are not many ways to press it by mistake. Also, I think 2 seconds is a long enough time to prevent accidental resets during installation. Therefore, I would prefer this to be reverted back.